lab-jacob/gulpfile.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
```

Comment:
This looks like a great start but since I can't run your gulp file 3 points are being deducted.